Doc/whatsnew/3.12.rst

```diff
@@ -413,6 +413,11 @@ Deprecated
   is no current event loop set and it decides to create one.
   (Contributed by Serhiy Storchaka and Guido van Rossum in :gh:`100160`.)
 
+* The :mod:`xml.etree.ElementTree` module now emits :exc:`DeprecationWarning`
```

Comment:
There should also be documentation in the docs for `xml.etree.ElementTree`. Those docs should say when the deprecation is scheduled to be executed (3.14 I think) and what people should do instead when they encounter the deprecation in their code.